### PR TITLE
Add blueprints to add default dependencies

### PR DESCRIPTION
**Pull request by jkusa**
Resolves #

## Description

Add blueprints to add default dependencies

## Proposed Changes

Needs #561

## Screenshots

n/a 

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/reports/blueprints/navi-reports/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  normalizeEntityName() {}, // no-op since we're just adding dependencies
+
+  afterInstall() {
+    return this.addAddonsToProject({
+      packages: [{ name: 'navi-core' }]
```
Comment (mattgraham1995) [inline packages/reports/blueprints/navi-reports/index.js]: Is this to force the core blueprint to run when running reports?

File: packages/core/blueprints/navi-core/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  normalizeEntityName() {}, // no-op since we're just adding dependencies
+
+  afterInstall() {
+    return this.addAddonsToProject({
```
Comment (kevinhinterlong) [inline packages/core/blueprints/navi-core/index.js]: Do addons typically print any messages here? Not necessary but could be nice

File: packages/reports/blueprints/navi-reports/index.js (hunk 1 above)
Comment (jkusa) [reply packages/reports/blueprints/navi-reports/index.js]: This allows you to simply do `ember install navi-reports` and get navi-core.

File: packages/core/blueprints/navi-core/index.js (hunk 2 above)
Comment (jkusa) [reply packages/core/blueprints/navi-core/index.js]: `addAddonsToProject` will print the addons it adds. Did you have something else in mind?

Comment (kevinhinterlong) [reply packages/core/blueprints/navi-core/index.js]: Nope just wanted to check it was printed somewhere

**Pull request merged**